src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.html

```diff
@@ -1,38 +1,40 @@
-            <p-checkbox class="dot-edit-layout__toolbar-save-template" label="{{ dotMessageService.get('editpage.layout.toolbar.save.template') }}" *ngIf="editTemplate && isLayout() && pageState.canCreateTemplate" binary="true" (onChange)="saveAsTemplateHandleChange($event);"></p-checkbox>
-            <button pButton secondary tiny type="button" (click)="onCancel($event)" label="{{ dotMessageService.get('dot.common.cancel') }}" class="dot-edit-layout__toolbar-action-cancel"></button>
-            <button pButton tiny type="submit" [disabled]="!isModelUpdated || !form.valid" label="{{ dotMessageService.get('editpage.layout.toolbar.action.save') }}" class="dot-edit-layout__toolbar-save"></button>
+            <input #templateName pInputText type="text" class="dot-edit-layout__toolbar-template-name" placeholder="{{ dotMessageService.get('editpage.layout.toolbar.template.name') }} *"
+                *ngIf="saveAsTemplate" formControlName="title">
+            <p-checkbox class="dot-edit-layout__toolbar-save-template" label="{{ dotMessageService.get('editpage.layout.toolbar.save.template') }}"
+                *ngIf="editTemplate && isLayout() && pageState.canCreateTemplate" binary="true" (onChange)="saveAsTemplateHandleChange($event);"></p-checkbox>
+            <button pButton secondary tiny type="button" (click)="onCancel($event)" label="{{ dotMessageService.get('dot.common.cancel') }}"
+                class="dot-edit-layout__toolbar-action-cancel"></button>
+            <button pButton tiny type="submit" [disabled]="!isModelUpdated || !form.valid" label="{{ dotMessageService.get('editpage.layout.toolbar.action.save') }}"
+                class="dot-edit-layout__toolbar-save"></button>
             <dot-template-addtional-actions-menu *ngIf="!isLayout()" [inode]="pageState.template.inode"></dot-template-addtional-actions-menu>
         </div>
     </p-toolbar>
 
     <div class="dot-edit-layout__actions">
         <dot-edit-page-info [pageState]="pageState"></dot-edit-page-info>
-
         <div class="dot-edit__layout-actions-right">
-            <button
-                pButton
-                tiny
-                link
-                type="button"
-                class="dot-edit-layout__toolbar-action-themes"
-                icon="fa-paint-brush"
-                label="{{ currentTheme?.name }}"
-                (click)="themeDialogVisibility = !themeDialogVisibility">
-            </button>
+            <div *ngIf="!currentTheme;then permission else noPermission "></div>
+            <ng-template #permission>
+                <div class="basic-container" pTooltip="{{ dotMessageService.get('org.dotcms.frontend.content.submission.not.proper.permissions') }}" tooltipPosition="left">
+                    <button pButton tiny link type="button" class="dot-edit-layout__toolbar-action-themes" icon="fa-paint-brush" label="{{ 'Themes' }}"
```

Comment:
Shoulnd't this label be translated? also if you passing a param that is just a strong, you don't need all this:

```label="{{ 'Themes' }}"```

you can just do:

```label="Themes"```